mtp_noms_ops/templates/security/accept_or_reject_check.html

```diff
@@ -273,10 +270,7 @@ <h2 class="govuk-heading-m">{% trans 'Decision' %}</h2>
         </p>
         <p>
           <strong>{% trans 'Decision details' %}:</strong>
-          {% for reason, value in check.rejection_reasons.items %}
-            {{ reason | get_human_readable_check_field }}{% if value != True %}: {{ value }}{% endif %}
-            <br/>
-          {% endfor %}
+          {{ check|format_security_check_rejection_reasons }}
```

Comment:
Neat! Question, would the following be equivalent?

```python
{{ format_security_check_rejection_reasons(check) }}
```

Reply:
that won't work, django template syntax does not allow calling functions like that. that's jinja style

Reply:
Cool, thanks, was just curious 👌